src/PKSim.Infrastructure/Services/ILoggerCreator.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PKSim.Infrastructure.Services
```

Comment:
That really looks like something that should be in core

Reply:
@msevestre @georgeDaskalakis I agree. We should move it when we are able to create new versions of our nugets so we could use them in MoBi as well.